### PR TITLE
Add packages

### PR DESCRIPTION
**Pull request by kenneth-Q**
Add gflags and hiptext
hiptext needs gflags.

### CODE REVIEW

File: packages/hiptext/font.cc.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+--- hiptext-0.2/src/font.cc	2019-03-03 13:47:56.235315625 +0800
++++ hiptext-0.2_mod/src/font.cc	2019-03-03 13:56:39.610971934 +0800
+@@ -11,7 +11,7 @@
+ #include "hiptext/graphic.h"
+ #include "hiptext/pixel.h"
+ 
+-DEFINE_string(font, "DejaVuSansMono.ttf",
++DEFINE_string(font, "/data/data/com.termux/files/usr/share/hiptext/DejaVuSansMono.ttf",
```
Comment (Grimler91) [inline packages/hiptext/font.cc.patch]: You can replace /data/data/com.termux/files/usr with @TERMUX_PREFIX@, it is automatically sed'ed when building: https://github.com/termux/termux-packages/blob/aa3bca1c11e6bb3ba5c16e8f4fb411af26ad8c94/scripts/build/termux_step_patch_package.sh#L10

Comment (kenneth-Q) [reply packages/hiptext/font.cc.patch]: Ok, I will do it later.
Thanks

**Comment by xeffyr** [comment]
Package "hiptext" is now available.

You can obtain it with these commands:
```
pkg install unstable-repo
pkg install hiptext
```

But "gflags" should be fixed as it's library is not getting installed to package (static build ?).

**Comment by kenneth-Q** [comment]
`
-- Install configuration: "MinSizeRel"
-- Installing: /data/data/com.termux/files/usr/lib/libgflags.a
-- Installing: /data/data/com.termux/files/usr/lib/libgflags_nothreads.a
-- Installing: /data/data/com.termux/files/usr/include/gflags/gflags.h
-- Installing: /data/data/com.termux/files/usr/include/gflags/gflags_declare.h
-- Installing: /data/data/com.termux/files/usr/include/gflags/gflags_completions.h
-- Installing: /data/data/com.termux/files/usr/include/gflags/gflags_gflags.h
-- Installing: /data/data/com.termux/files/usr/lib/cmake/gflags/gflags-config.cmake
-- Installing: /data/data/com.termux/files/usr/lib/cmake/gflags/gflags-config-version.cmake
-- Installing: /data/data/com.termux/files/usr/lib/cmake/gflags/gflags-targets.cmake
-- Installing: /data/data/com.termux/files/usr/lib/cmake/gflags/gflags-targets-minsizerel.cmake
-- Installing: /data/data/com.termux/files/usr/lib/cmake/gflags/gflags-nonamespace-targets.cmake
-- Installing: /data/data/com.termux/files/usr/lib/cmake/gflags/gflags-nonamespace-targets-minsizerel.cmake
-- Installing: /data/data/com.termux/files/usr/bin/gflags_completions.sh
-- Installing: /data/data/com.termux/files/usr/lib/pkgconfig/gflags.pc
-- Installing: /home/builder/.cmake/packages/gflags/bfd1b9fbe15a2d7caccacf6b9f884ea0
termux - build of 'gflags' done
`
From my building log, I got these information. Path: usr/lib/; usr/include/gflags/ should be installed.
Let me have a look.

**Comment by kenneth-Q** [comment]
You are right, it is static build now.
I will fix it later.

**Pull request merged**